account/identity.go

```diff
@@ -57,6 +63,14 @@ func (u NullUUID) Value() (driver.Value, error) {
 	return u.UUID.Value()
 }
 
+// This struct is used to return the Organizations for which an Identity is associated
+type IdentityOrganization struct {
```

Comment:
Wondering if we should move out the authorization/organization related methods/structs to a different go file in the same package ? Saying this, so that we can keep `account/identity.go` clean and simple, and move out the complex derivatives to smaller/logical files into something like `account/organization.go`

Reply:
I thought about this, however since this struct is used nowhere else except for a single method in the identity.go file, it didn't make much sense to me to move it.

Reply:
I was thinking about moving this and the `ListOrganizations(...)` method, what do you think?

Reply:
How about we create a new file called identity_management.go and put all IdM related functions there?  Would the ListOrganizations() etc methods still be provided by IdentityRepository, or are you thinking they should be moved to a new Repository class (IdentityManagementRepository?) instead?

Reply:
I'm not in favor of making a new repository class :) We could just move the one struct and the one repository method into the different file for slightly better code organization - not really considering moving that to a different `interface implementation `   at the moment.

Reply:
No problem, I'll just move the struct and ListOrganizations() method to a new file, but keep them in the existing IdentityRepository.

Reply:
These have been moved now to `account/identity_management.go`